studies/0063_auto_20200428_1253.py

```diff
@@ -0,0 +1,146 @@
+# Generated by Django 2.0 on 2020-04-28 16:53
```

Comment:
OK, looking at these, I don't think any of the migrations have to do with the upgrade to django itself - these are probably just field alterations that are happening now because we didn't migrate the last time we edited model fields (I think that might have been your last PR, which was modifying `on_delete` behavior).

As such, when I merge, I'm probably going to leave out these migrations (unless you can think of a good reason not to)

Reply:
Oh, that makes sense. (And I guess I don't understand migrations well enough, because now I'm surprised deploying those changes actually modified the behavior of deletion.)

If they ought to have happened along with the previous PR, though, why wait on applying them?

Some of the operations are actually related to the Django upgrade - changing default values so they're not {}, and something that got created about changing a manager:

```
  operations = [
        migrations.AlterModelOptions(
            name="response",
            options={
                "base_manager_name": "related_manager",
                "ordering": ["-demographic_snapshot__created_at"],
                "permissions": (
                    (
                        "view_all_response_data_in_analytics",
                        "View all response data in analytics",
                    ),
                ),
            },
        ),
        migrations.AlterField(
            model_name="study",
            name="metadata",
            field=project.fields.datetime_aware_jsonfield.DateTimeAwareJSONField(
                default=dict
            ),
        ),
        migrations.AlterField(
            model_name="study",
            name="structure",
            field=project.fields.datetime_aware_jsonfield.DateTimeAwareJSONField(
                default=studies.models.default_study_structure
            ),
        ),
        migrations.AlterField(
            model_name="studytype",
            name="configuration",
            field=project.fields.datetime_aware_jsonfield.DateTimeAwareJSONField(
                default=studies.models.default_configuration
            ),
        ),
    ]
```